File: Python 3/561_ ArrayPartitionI.py
class Solution:
    def arrayPairSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        result = 0
        nums.sort()

        for i in nums[::2]:
            result += i

        return result

    def arrayPairSum2(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        return sum(sorted(nums)[::2])

File: Java/561_ ArrayPartitionI.java
// 140 ms
class Solution {
    public int arrayPairSum(int[] nums) {
        Arrays.sort(nums);
        
        return IntStream.range(0, nums.length)
                    .filter(n -> n % 2 == 0)
                    .map(i -> nums[i])
                    .sum();
    }
}